### PR TITLE
Assets Management

### PR DESCRIPTION
**Pull request by geronimo-iia**
Hello,
in this pull request, you could retreive:
- minor fix on package version
- fix on colors/green missing methods
- assets managements
- doc in readme, a little how to,
- updated brume.yaml

Note: if you did not specifiy "assets" in configuration, this feature do nothing.

Let'm known if I need to change something

Jerome

### CODE REVIEW

**Comment by geronimo-iia** [comment]
@flou first conflict resolution with github GUI, nice to see in action

File: brume/cli.py [hunk 1]
```diff
@@ -7,12 +7,21 @@
...
 from stack import Stack
 from template import Template
-
+from assets import sendAssets
 conf = Config.load('brume.yml')
 templates_config = conf['templates']
 cf_config = conf['stack']
 
 
+def processAssets():
```
Comment (flou) [inline brume/cli.py]: I think we need to use snake_case for method names.

File: brume/assets.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import sys
+import os
+import boto3
+import click
+
+s3_client = boto3.client('s3')
+
+def sendAssets(local_path, s3_bucket , s3_path= ''):
```
Comment (flou) [inline brume/assets.py]: I think we need to use snake_case for method names.

File: brume/cli.py (hunk 1 above)
Comment (geronimo-iia) [inline brume/cli.py]: I'm not a good fan of _ in method name, but I'll fix that

File: README.rst [hunk 3]
```diff
@@ -73,6 +73,19 @@ If you use ``brume upload``, you need to tell brume where the templates are and
...
 Given the above configuration and if you have a ``Main.cform`` in ``project/cfn``, the template would be uploaded to ``https://my-bucket.s3.amazonaws.com/assets/cloudformation/Main.cform``.
 
+Assets
+~~~~~~
+
+If 'assets' configuration is present you can send additionnal resource to target s3 URI (like user data script, application config file, ...).
+
+In your template, you can build assets url like this:
+
+```
```
Comment (geronimo-iia) [inline README.rst]: oups :)

**Comment by geronimo-iia** [comment]
@flou you could see requested change

**Pull request merged**